modules/api_test/api_test.install

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Creates a OAuth2 Client with the Page Creator role scope and a related user.
+ */
+
+use Drupal\user\Entity\User;
+use Drupal\simple_oauth\Entity\Oauth2Client;
+
+/**
+ * Implements hook_install().
+ */
+function api_test_install() {
+  // Create a sample user to use with the client.
+  $edit = [
+    'name' => 'api-test-user',
+    'mail' => 'api-test-user@example.com',
+    'pass' => 'admin',
+    'status' => 1,
```

Comment:
Rather than pass a boolean for status, let's call $account->activate() instead to keep the abstraction sealed. :)